tests/model_serving/model_server/model_car/conftest.py

```diff
@@ -18,14 +18,16 @@ def model_car_serverless_inference_service(
     unprivileged_model_namespace: Namespace,
     serving_runtime_from_template: ServingRuntime,
 ) -> Generator[InferenceService, Any, Any]:
+    deployment_mode = request.param.get("deployment-mode", KServeDeploymentType.SERVERLESS)
     with create_isvc(
         client=unprivileged_client,
-        name="serverless-model-car",
+        name=f"model-car-{deployment_mode.lower()}",
```

Comment:
nitpick: With this change, I think the name of the fixture should be subject for renaming.